rogue-thi-app/pages/personal.js

```diff
@@ -165,7 +165,9 @@ export default function Personal () {
               <span className="text-muted">
                 {ects !== null && ects + ' ECTS'}
                 {average && ' · '}
-                {average && '∅ ' + average}
+                {average && '∅ ' + average.result}
+                {average && 'missingWeight' in average && average.missingWeight === 1 && ' (' + average.missingWeight + ' Gewichtung fehlt)'}
+                {average && 'missingWeight' in average && average.missingWeight > 0 && ' (' + average.missingWeight + ' Gewichtungen fehlen)'}
```

Comment:
If `missingWeight` is `1` both messages are displayed(?)

Is the `'missingWeight' in average` needed? I do not think the `loadGradeAverage` function ever returns an object without the field.
If i am mistaken i think `average?.missingWeight === 1` and `average?.missingWeight > 1` would be the best idea.

Reply:
Absolutely right, I missed that when adding the extra case for only one missingWeight. I'll change that.

Reply:
You are right, the object always has that key to we can skip that.

I also missed that case when adding the extra case for only one `missingWeight` afterwards. I'll change that :)